package/kernel/linux/modules/multimedia.mk

```diff
@@ -0,0 +1,857 @@
+MMEDIA_MENU:=Multimedia
+
+define KernelPackage/multimedia
+  SUBMENU:=$(MMEDIA_MENU)
+  TITLE:=Multimedia
+  KCONFIG:= \
+	CONFIG_MEDIA_SUPPORT=m \
+	CONFIG_MEDIA_SUPPORT_FILTER=y \
+	CONFIG_MEDIA_USB_SUPPORT=y
+  DEPENDS:=@USB_SUPPORT
+  FILES:=$(LINUX_DIR)/drivers/media/mc/mc.ko
+  AUTOLOAD:=$(call AutoLoad,80,mc)
+endef
+define KernelPackage/multimedia/description
+  Kernel module for Multimedia support (cameras, TV tuners)
+endef
+$(eval $(call KernelPackage,multimedia))
+
+# DVB & tuners
+
+define KernelPackage/dvb-usb-v2
+  SUBMENU:=$(MMEDIA_MENU)
+  TITLE:=USB DVB support
+  KCONFIG:= \
+	CONFIG_MEDIA_DIGITAL_TV_SUPPORT=y \
+	CONFIG_MEDIA_ANALOG_TV_SUPPORT=y \
+	CONFIG_DVB_CORE=m \
+	CONFIG_DVB_USB_V2=m
+  DEPENDS:=+kmod-i2c-core +kmod-usb-core kmod-multimedia
+  FILES:= \
+	$(LINUX_DIR)/drivers/media/dvb-core/dvb-core.ko \
+	$(LINUX_DIR)/drivers/media/usb/dvb-usb-v2/dvb_usb_v2.ko
```

Comment:
Need to add:
$(LINUX_DIR)/drivers/media/rc/rc-core.ko

Reply:
Did you encounter any error?

I know that Astrometa has an IR receiver, but remotes should be a separate package. Most users won't need a remote for their router.

Reply:
> Did you encounter any error?
> 
> I know that Astrometa has an IR receiver, but remotes should be a separate package. Most users won't need a remote for their router.

x86_64 compilation error, need rc-core.ko to compile successfully.

Reply:
I can't reproduce. I need your .config to try it myself.

AFAIK, there's no dependency in kernel menuconfig between dvb and rc.
Also, your proposed change, `$(LINUX_DIR)/drivers/media/rc/rc-core.ko`, only installs the kernel module after it was successfully built. It doesn't make any sense.